package.json

```
@@ -16,7 +16,7 @@
     "lint:fix": "fedx-scripts eslint --fix --ext .js --ext .jsx .",
     "snapshot": "fedx-scripts jest --updateSnapshot",
     "start": "fedx-scripts webpack-dev-server --progress",
-    "test": "fedx-scripts jest --coverage --passWithNoTests"
+    "test": "fedx-scripts jest --coverage --passWithNoTests --watch"
```

```suggestion
    "test": "fedx-scripts jest --coverage --passWithNoTests"
    "test:watch": "fedx-scripts jest --coverage --passWithNoTests --watch"
```
Please add a different script for that, the test script is used in git checks and the image builder and breaks them